pydy/system.py

```diff
@@ -545,7 +578,15 @@ def _Kane_constant_symbols(self):
         TODO temporary.
 
         """
-        constants = list(self.eom_method._find_othersymbols(
-            *self._Kane_inlist_insyms()))
-        constants.remove(sym.symbols('t'))
+        from_eoms, from_sym_lists = self._Kane_inlist_insyms()
+        if cmp(parse_version(SYMPY_VERSION), parse_version('0.7.6')) > -1:
```

Comment:
Yeah, I had something like that in my first shot and then changed it to this because it seemed simpler. I guess the function would be either `greater_than_076()` or `less_than_076`. But if we have new changes in SymPy 0.7,7, then I will need new functions for picking out the version intervals. I guess I thought this was better than starting down that road.